Updated Quick Start Guide (#25)
hms-networks/flexy-cumulocity-connector: QUICK-START.md

The Ewon Flexy Cumulocity Connector package provides a connector-based solution to Cumulocity for
linking Ewon devices using a direct data path with a Flexy Java application.

> **Note**: This document provides a quick-start guide to use this connector, for a more comprehensive description of this process and the connector, please check out the [Installation and Configuration Guide](https://hmsnetworks.blob.core.windows.net/www/docs/librariesprovider10/downloads-monitored/software/flexy-cumulocity-integration-installation-and-configuration-guide-v1.0.pdf?sfvrsn=1490cbd7_5) and the [Technical Documentation](https://hmsnetworks.blob.core.windows.net/www/docs/librariesprovider10/downloads-monitored/software/solution-pack---ewon-cumulocity-connector---technical-documentation.pdf).

Reviewer: ```suggestion
**Note**: This document provides a quick start guide for using this connector. For a more comprehensive description of this process and the connector, please check out the [Installation and Configuration Guide](https://hmsnetworks.blob.core.windows.net/www/docs/librariesprovider10/downloads-monitored/software/flexy-cumulocity-integration-installation-and-configuration-guide-v1.0.pdf?sfvrsn=1490cbd7_5) and the [Technical Documentation](https://hmsnetworks.blob.core.windows.net/www/docs/librariesprovider10/downloads-monitored/software/solution-pack---ewon-cumulocity-connector---technical-documentation.pdf).
```

Populate the `Host`, `BootstrapUsername`, `BootstrapPassword`, and `BootstrapTenant` fields with the
corresponding values for your Cumulocity tenant. The `Host` field should be populated with the
MQTT broker's host for your Cumulocity instance, for example, `mqtt.eu-latest.cumulocity.com` or

> `mqtt.us-latest.cumulocity.com`. Do not include the scheme or port as they are automatically

Reviewer: ```suggestion
`mqtt.us-latest.cumulocity.com`. Do not include the scheme or port as they are automatically 
```

populated `CumulocityConnectorConfig.json` from earlier.

1. `flexy-cumulocity-connector-{VERSION}-full.jar`

>    1. Located in the `target` folder

Reviewer: ```suggestion
    1. Located in the `target` folder
```

2. `jvmrun`

>    1. Located in the `starting-files` folder

Reviewer: ```suggestion
    1. Located in the `starting-files` folder
```

3. `CumulocityConnectorConfig.json`

>    1. Located in the `starting-files` folder

Reviewer: ```suggestion
    1. Located in the `starting-files` folder
```
